src/network/networkd-network.c

```diff
@@ -1416,6 +1417,39 @@ int config_parse_dhcp_route_table(const char *unit,
         return 0;
 }
 
+int config_parse_ipv6_mtu(
+                const char *unit,
+                const char *filename,
+                unsigned line,
+                const char *section,
+                unsigned section_line,
+                const char *lvalue,
+                int ltype,
+                const char *rvalue,
+                void *data,
+                void *userdata) {
+
+        int *mtu = data;
+        int k, r;
+
+        assert(filename);
+        assert(lvalue);
+        assert(rvalue);
+
+        r = config_parse_int(unit, filename, line, section, section_line, lvalue, ltype, rvalue, &k, userdata);
+        if (r < 0)
```

Comment:
Unfortunately this is not going to work because `config_parse_int` does not report an error properly. It always returns 0.